v0/destinations/custify/data/CUSTIFYTrackConfig.json

```diff
@@ -1,30 +1,14 @@
 [
   {
     "destKey": "user_id",
-    "sourceKeys": [
-      "userId",
-      "traits.userId",
-      "traits.id",
-      "context.traits.userId",
-      "context.traits.id"
-    ],
-    "required": false
-  },
-  {
-    "destKey": "company_id",
-    "sourceKeys": [
-      "groupId",
-      "traits.groupId",
-      "context.traits.groupId"
-    ],
```

Comment:
Don't we need to send `conpany_id` in track calls?

Reply:
Yeah we need along with these due merge it didnt came